src/v/cloud_storage/partition_probe.cc

```diff
@@ -42,79 +42,82 @@ partition_probe::partition_probe(const model::ntp& ntp) {
           "read_bytes",
           [this] { return _bytes_read; },
           sm::description("Total bytes read from remote partition"),
-          labels),
+          partition_labels),
         sm::make_counter(
           "read_records",
           [this] { return _records_read; },
           sm::description("Total number of records read from remote partition"),
-          labels),
+          partition_labels),
 
+        sm::make_gauge(
+          "chunk_size",
+          [this] { return _chunk_size; },
+          sm::description("Size of chunk downloaded from cloud storage"),
+          partition_labels),
+      });
```

Comment:
Should we always aggregate by shard for `partition_probe`? I don't think it's much use knowing which shard read the bytes.

Reply:
done